waltz-storage/src/main/resources/log4j.xml

```diff
@@ -7,7 +7,7 @@
         <param name="Target" value="System.out" />
         <param name="Encoding" value="UTF-8" />
         <layout class="org.apache.log4j.PatternLayout">
-            <param name="ConversionPattern" value="%d{yyyy-MM-dd HH:mm:ss.SSS} UTC %-5p: %c - %m%n" />
+            <param name="ConversionPattern" value="[%t]%d{yyyy-MM-dd HH:mm:ss.SSS} UTC %-5p: %c - %m%n" />
```

Comment:
Unrelated to what this PR is trying to solve...
Just a generic change to help in better logging on local clusters...